### ISSUE TITLE
Linux 5.6: failed to compile

### ISSUE THREAD

**Issue by k0ste** [aabc/ipt-ratelimit#17]
```shell
make -C /usr/lib/modules/5.6.2-arch1-2/build M=/home/k0ste/sandbox/AUR/ipt_ratelimit/src/ipt-ratelimit-0.2 modules CONFIG_DEBUG_INFO=y
make[1]: Entering directory '/usr/lib/modules/5.6.2-arch1-2/build'
  CC [M]  /home/k0ste/sandbox/AUR/ipt_ratelimit/src/ipt-ratelimit-0.2/xt_ratelimit.o
/home/k0ste/sandbox/AUR/ipt_ratelimit/src/ipt-ratelimit-0.2/xt_ratelimit.c: In function ‘htable_create’:
/home/k0ste/sandbox/AUR/ipt_ratelimit/src/ipt-ratelimit-0.2/xt_ratelimit.c:704:7: error: passing argument 4 of ‘proc_create_data’ from incompatible pointer type [-Werror=incompatible-pointer-types]
  704 |       &ratelimit_fops, ht);
      |       ^~~~~~~~~~~~~~~
      |       |
      |       const struct file_operations *
In file included from /home/k0ste/sandbox/AUR/ipt_ratelimit/src/ipt-ratelimit-0.2/xt_ratelimit.c:29:
./include/linux/proc_fs.h:59:31: note: expected ‘const struct proc_ops *’ but argument is of type ‘const struct file_operations *’
   59 | extern struct proc_dir_entry *proc_create_data(const char *, umode_t,
      |                               ^~~~~~~~~~~~~~~~
cc1: some warnings being treated as errors
make[2]: *** [scripts/Makefile.build:268: /home/k0ste/sandbox/AUR/ipt_ratelimit/src/ipt-ratelimit-0.2/xt_ratelimit.o] Error 1
make[1]: *** [Makefile:1683: /home/k0ste/sandbox/AUR/ipt_ratelimit/src/ipt-ratelimit-0.2] Error 2
make[1]: Leaving directory '/usr/lib/modules/5.6.2-arch1-2/build'
make: *** [Makefile:12: xt_ratelimit.ko] Error 2
```

**Comment by aabc** [comment]
Ouch, I pushed an incomplete fix. Wait a bit before testing.

**Comment by aabc** [comment]
OK, it should be properly fixed by now.

**Comment by k0ste** [comment]
Thanks. v0.3 release?

**Comment by aabc** [comment]
Good idea.

**Comment by aabc** [comment]
Hm, 0.3 is released 2017-05-10. There are only today's commits for 0.4...

**Comment by aabc** [comment]
Ok, we can mark it as 0.3.1, probably...

**Comment by k0ste** [comment]
Oh, yes. I mean github release exactly.

**Comment by aabc** [comment]
Added missing v0.3 tag, and released v0.3.1.
